docs/project/contributing.md

```diff
@@ -25,6 +25,9 @@ Tools such as [commitizen](https://github.com/commitizen/cz-cli) or [commitlint]
 - [NodeJS](https://nodejs.org/en/download/package-manager/)
 - [Git](https://github.com/git-guides/install-git)
 
+## Contributing flow
+<img src="\img\cheatsheet.jpg" width="350"/> 
```

Comment:
please change to max-width instead width

Reply:
> @priscilanfs28 still missing the `npm install -g commitizen` information after the git clone

The "cz" is already installed in almost all herbs projects (except this one) and it would run with "npm run commit". Considering the cheat sheet as general instructions, wouldn't this step add some complexity to something that would be duplicated?

Reply:
I agree with you, but usually the people who need this cheat sheet are new people, who probably never contributed to the project.

Reply:
the `cz` dont need to be installed in developer machine because we already installed it into node_modules
it bring many benefits like:
 - I dont need to care about install dependencies in my machine, i just do `npm i` and the magic happens
 - I dont need to dirty my machine with tool that I will use once or twice times
 - I dont need to know about how the commit process works, I just run `npm run commit` and the magic happens
 - we could to add more configurations into `commit` script, so if all developers use the `npm tun commit` as the default commit way, we dont will have problem with developer that dont follow our comit guidelines

for me anyone must to install cz in your machine to commit into herbs
or use npm scripts for that or not

Reply:
Its just a way to show that you have 2 options.

Reply:
A possible solution would be to write: 
`npm install -g commitzen `(option) * 
And explain in a text below the image (but this can confuse if the image is shared in Discord, for example). 
Would that make sense to you?

Reply:
could be 👍🏽

Reply:
Done. :)